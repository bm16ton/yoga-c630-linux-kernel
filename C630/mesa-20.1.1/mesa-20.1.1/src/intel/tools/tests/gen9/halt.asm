(-f0.1.any4h) halt(8) JIP: 176  UIP: 192                        { align1 1Q };
halt(8)         JIP: 16         UIP: 16                         { align1 1Q };
(-f0.1.any4h) halt(16) JIP: 176 UIP: 192                        { align1 1H };
halt(16)        JIP: 16         UIP: 16                         { align1 1H };
