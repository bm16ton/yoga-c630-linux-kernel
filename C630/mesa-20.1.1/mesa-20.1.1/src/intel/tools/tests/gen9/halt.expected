2a 00 76 00 21 0e 00 20 c0 00 00 00 b0 00 00 00
2a 00 60 00 20 0e 00 20 10 00 00 00 10 00 00 00
2a 00 96 00 21 0e 00 20 c0 00 00 00 b0 00 00 00
2a 00 80 00 20 0e 00 20 10 00 00 00 10 00 00 00
